src/kernels/jupyter/types.ts

```diff
@@ -235,9 +235,9 @@ export interface IJupyterServerUriStorage {
     getSavedUriList(): Promise<{ uri: string; time: number; displayName?: string }[]>;
     removeUri(uri: string): Promise<void>;
     clearUriList(): Promise<void>;
-    getUri(): Promise<string>;
     getRemoteUri(): Promise<string | undefined>;
-    setUri(uri: string): Promise<void>;
+    setUriToLocal(): Promise<void>;
```

Comment:
Added these two methods so that the uri string used to determien whether its local or remote isn't exposed elsewhere.